test/integrations/destinations/mp/common.ts

```diff
@@ -5,7 +5,7 @@ const defaultMockFns = () => {
 const sampleDestination = {
   Config: {
     apiKey: 'dummyApiKey',
-    token: 'dummyApiKey',
+    token: 'test_api_token',
```

Comment:
Why are you updating this value?

Reply:
making it consistent with all other mixpanel testacases.